crates/bevy_text/src/pipeline.rs

```diff
@@ -18,12 +18,15 @@ use crate::{
 /// A wrapper around a [`cosmic_text::FontSystem`]
 struct CosmicFontSystem(cosmic_text::FontSystem);
 
-impl Default for CosmicFontSystem {
-    fn default() -> Self {
+impl CosmicFontSystem {
+    pub fn new(load_system_fonts: bool) -> Self {
         let locale = sys_locale::get_locale().unwrap_or_else(|| String::from("en-US"));
-        let db = cosmic_text::fontdb::Database::new();
-        // TODO: consider using `cosmic_text::FontSystem::new()` (load system fonts by default)
-        Self(cosmic_text::FontSystem::new_with_locale_and_db(locale, db))
+        let mut db = cosmic_text::fontdb::Database::new();
+        if load_system_fonts {
+            db.load_system_fonts();
```

Comment:
I assume this just noops on Wasm and Android? Or what happens there? Do we get a warning?